.github/actions/promote_docker_image/test/action.test.ts

```diff
@@ -0,0 +1,33 @@
+import { MockGithub } from '@kie/mock-github';
```

Comment:
So as this test shows, there's nothing to really test here. In this case I think we'd be better served deleting this tests as it really offers no value.

Reply:
Alright cool I just left in in place in case we wanted to extend it later but may as well just remove it altogether.